pkg/environment/environment.go

```diff
@@ -0,0 +1,25 @@
+package environment
```

Comment:
I won't block this PR on this, but this is wrong. It keeps functions in a single module grouped by the function they use, not by their "unit". That makes this into a junkyard of env variable handling.

This points at the issue that we are coupling everything together here. We should have completely separated lane for every of these modes (operator, handler, webhook). We should read the parameter (IsOperator) in main and just run the right component. Just ranting, I think we will need to make some order in the code-base, sooner or later.

Reply:
At the follow-up PR we will refactor this.